rollup.config.js

```diff
@@ -33,12 +33,16 @@ function getPlugins(standalone, transpile = true) {
 			)
 		}),
 		transpile && babel({
-			extensions: [
+			babelHelpers:       'runtime',
+			// erring otherwise in attempt to find `@babel/plugin-transform-runtime`
+			//   added by `babel-preset-trigen`; see
+			//   https://github.com/rollup/plugins/issues/381
+			skipPreflightCheck: true,
+			extensions:         [
```

Comment:
to make diff cleaner, please move extensions option to top of object

and then I approve it

Reply:
Ok, sure. Added.

Btw, do you happen to know--is the code coverage really as low as 1.39% or is there some bug in jest/jest configuration?

Reply:
@brettz9 
> Coverage remained the same at 78.705%

Reply:
Cool... But how about what shows on the command line:

![image](https://user-images.githubusercontent.com/20234/81640802-17751680-9452-11ea-8e95-2277b20394e1.png)

Reply:
@brettz9 screenshot testing with browsers starts only on linux (ci) due to font rendering/etc. issues.